### PR TITLE
Ryan/azure fixes

### PR DESCRIPTION
**Pull request by ryancragun**
Use Bash because Dash is ridiculous

### CODE REVIEW

File: files/chef-marketplace-cookbooks/chef-marketplace/recipes/_publishing_enable.rb [hunk 1]
```diff
@@ -75,3 +75,8 @@
     ) && mirrors_reachable?
   end
 end
+
+package 'walinuxagent' do
+  action [:install, :upgrade]
```
Comment (adamleff) [inline files/chef-marketplace-cookbooks/chef-marketplace/recipes/_publishing_enable.rb]: Just `:upgrade` would do the trick, no?

Comment (ryancragun) [inline files/chef-marketplace-cookbooks/chef-marketplace/recipes/_publishing_enable.rb]: Nope, it's a held package so we have to it that way.

Comment (adamleff) [inline files/chef-marketplace-cookbooks/chef-marketplace/recipes/_publishing_enable.rb]: What does "held package" mean in this context?

**Comment by adamleff** [comment]
:+1:

**Comment by andrewelizondo** [comment]
:+1: LGTM

**Pull request merged**